float64 mcu_time
float64 ax
float64 ay
float64 az
float64 wx
float64 wy
float64 wz
float64 wz_fog
float64 odo_speed
float64 odo_time
float64 temp